### PR TITLE
Create tidb-binlog.md

### PR DESCRIPTION
**Pull request by lilin90**
PTAL

### CODE REVIEW

File: op-guide/tidb-binlog.md [hunk 1]
```diff
@@ -0,0 +1,365 @@
...
+<!-- /TOC -->
+
+
+## About TiDB-Binlog
+
+TiDB-Binlog is a tool for enterprise users to collect binlog files for TiDB and provide real-time backup and synchronization.
+
+TiDB-Binlog supports the following scenarios:  
+
+- *Data synchronization*: to synchronize TiDB cluster data to other databases
```
Comment (queenypingcap) [inline op-guide/tidb-binlog.md]: change "*" to "**"

File: op-guide/tidb-binlog.md [hunk 2]
```diff
@@ -0,0 +1,365 @@
+---
+title: Deploy TiDB-Binlog
+category: advanced
+---
+
+<!-- TOC -->
```
Comment (queenypingcap) [inline op-guide/tidb-binlog.md]: Please remove this line

File: op-guide/tidb-binlog.md (hunk 1 above)
Comment (lilin90) [reply op-guide/tidb-binlog.md]: Changed.

File: op-guide/tidb-binlog.md (hunk 2 above)
Comment (lilin90) [reply op-guide/tidb-binlog.md]: Removed.

**Pull request merged**